src/geode/model/helpers/convert_to_mesh.cpp

```diff
@@ -635,6 +962,57 @@ namespace geode
         return std::make_tuple( std::move( surface ), std::move( solid ) );
     }
 
+    std::tuple< std::unique_ptr< EdgedCurve< 2 > >, ModelToMeshMappings >
+        new_convert_section_into_curve( const Section& section )
+    {
+        return new_convert_model_into_curve( section );
+    }
+
+    std::tuple< std::unique_ptr< EdgedCurve< 3 > >, ModelToMeshMappings >
+        new_convert_brep_into_curve( const geode::BRep& brep )
+    {
+        return new_convert_model_into_curve( brep );
+    }
+
+    std::tuple< std::unique_ptr< SurfaceMesh3D >, ModelToMeshMappings >
+        new_convert_brep_into_surface( const BRep& brep )
+    {
+        return new_convert_model_into_surface( brep );
+    }
+
+    std::tuple< std::unique_ptr< SurfaceMesh2D >, ModelToMeshMappings >
+        new_convert_section_into_surface( const Section& section )
+    {
+        return new_convert_model_into_surface( section );
+    }
```

Comment:
I'd order the section functions before the brep ones to make searching easier later on